### PR TITLE
Weighted Kappa

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### Reference Issues/PRs
#243 
#### What does this implement/fix? Explain your changes.
- `weighted_kappa` method added

### CODE REVIEW

**Comment by sepandhaghighi** [comment]
@sadrasabouri @alirezazolanvari

Hi folks ;-)

This PR is for #243, please check carefully and give me some feedback ;-)
@alirezazolanvari please reformat added references (`70`,`71`) in `README.md` - `weighted_kappa` branch and then inform me.

**Notice 1 :** Do not hurry, we have enough time ;-)
**Notice 2 :** This PR has 2 reviewer, so please don't merge this PR to dev branch before finishing the discussion

Best Regards
Sepand Haghighi

**Comment by sepandhaghighi** [comment]
@alirezazolanvari `Document` references reformated.
If everything is okay, please merge this PR after approval

File: pycm/pycm_obj.py [hunk 1]
```diff
@@ -693,3 +694,25 @@ def weighted_average(self, param, weight=None, none_omit=False):
...
+        """
+        Calculate weighted kappa.
+
+        :param weight: weight matrix
+        :type weight: dict
+        :return: weighted kappa as float
+        """
+        if matrix_check(weight) is False:
+            warn(WEIGHTED_KAPPA_WARNING, RuntimeWarning)
+            return self.Kappa
```
Comment (alirezazolanvari) [inline pycm/pycm_obj.py:708]: Isn't it better to raise an error in the case of wrong weight parameter format?

Comment (sepandhaghighi) [reply pycm/pycm_obj.py:708]: You are right, raising an error is a good option but I think for parameters like `weight` here and `alpha` in `CI` method that have a well-known default value, **warning** is a **better** option ;-)

**Pull request merged**